### ISSUE TITLE
Cannot expand path hierarchy on some paths

### ISSUE THREAD

**Issue by gtfierro** [BTrDB/mr-plotter#12]
I've been trying to add a bunch of new data to BTrDB with collections like

```
Cory_Hall_Dent_Meters/basement-3/elt-B/A/apparent_pf
Cory_Hall_Dent_Meters/358/elt-A/A/true_energy
```

When I try to expand the `Cory_Hall_Dent_Meters` node in the plotter, I get the following error in the JS console.

```
Uncaught TypeError: currNodes.push is not a function
    at pathsToTree (streamtree.js:343)
    at Object.success (streamtree.js:138)
    at i (jquery-1.12.0.min.js:2)
    at Object.fireWith [as resolveWith] (jquery-1.12.0.min.js:2)
    at y (jquery-1.12.0.min.js:4)
    at XMLHttpRequest.c (jquery-1.12.0.min.js:4)
```

I think there's ~30 of these paths under the top level `Cory_Hall_Dent_Meters`.

This does not seem to affect any of the other paths. I'm happy to change something if it will help me avoid this issue

**Comment by samkumar** [comment]
Can you send me a link so I can try to reproduce this?

**Comment by gtfierro** [comment]
Go to https://plot.xbos.io/ and click on "Cory_Hall_Dent_Meters" after you log in

**Comment by samkumar** [comment]
I think I have fixed this in aa8bb1fb8640696398a188104a6b5a135e1948d2. Reopen the issue if problems persist.
